### ISSUE TITLE
Coelacanth does not eat krill

### ISSUE THREAD

**Issue by StevJubz** [JurassiCraftTeam/JurassiCraft2#223]
Coelacanths are piscivores, if for gameplay purposes why not replace krill with small generic fish schools? Coelacanth don't hunt fish so you can just leave the fish schools in place and not really have them swim around

**Comment by gegy1000** [comment]
I'm sure some cool stuff could be done with this for 1.14. Would be awesome to have them chase fish.

**Comment by gegy1000** [comment]
Oh. In that case, I don't understand the point of the suggestion here?

**Comment by StevJubz** [comment]
I think it's a flawed feature and I was suggesting krill be replaced with fish schools and someone else labeled this ass a suggestion

**Comment by gegy1000** [comment]
Yes, but why replace krill with fish schools? If fish schools serve no function, why not just remove the krill, and what's flawed about the current implementation?

**Comment by StevJubz** [comment]
they don't eat krill, they eat fish. I'm sugggesting that krill be replaced by a generic fish school that floats in the water

**Comment by Gamebrothers4** [comment]
Sorry but this sounds like a suggestion not a replacement

**Comment by Gamebrothers4** [comment]
Or a suggestion for a replacement

**Comment by StevJubz** [comment]
It's a suggestion to replace the krill with a generic fish school because coelacanth don't eat krill

**Comment by Gamebrothers4** [comment]
The coel does eat krill

**Comment by Codyrex78117** [comment]
He means they shouldn't eat krill.
